<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>27</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color:gray;
border: 1px solid gray;
color:#ffffff;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="div">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiply">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MemAdd">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MemClear">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Equals">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeSign">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MemGet">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:yellow;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subtract">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:red;
border: 1px solid gray;
padding: 5px;
}
QPushButton:pressed{
background-color:#A9A9A9;
border: 1px solid gray;
padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Calculator">
    <property name="title">
     <string>Calculator</string>
    </property>
   </widget>
   <addaction name="Calculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
